packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -625,20 +653,36 @@ export class CallContext {
     });
   }
 
-  public setDeviceManagerUnparentedView(
-    localVideoStream: LocalVideoStreamState,
-    view: VideoStreamRendererViewState | undefined
-  ): void {
+  public setDeviceManagerUnparentedView(args: {
+    localVideoStream: LocalVideoStreamState;
+    view: VideoStreamRendererViewState | undefined;
+    /* @conditional-compile-remove(video-background-effects) */
+    localVideoStreamEffectsAPI: VideoEffectsFeature;
+  }): void {
+    /* @conditional-compile-remove(video-background-effects) */
+    {
+      this._unparentedViewVideoEffectsSubscriber?.unsubscribe();
```

Comment:
We are invoking the unsubscribe here to just to make sure there isn't a unparented view already? is there anything stopping us from having more than one unparented view?

Reply:
Just a quick failsafe if setDeviceManagerUnparentedView is called twice in a row we don't set up multiple subscriptions